infra/deployments/hub/dev/iam.tf

```diff
@@ -13,6 +13,7 @@ locals {
   ]
```

Comment:
Is the goal to have Jonathan access the bucket in the GCS console directly, or what do we want to enable with this PR?